util/dvsim/FormalCfg.py

```diff
@@ -59,61 +59,61 @@ def parse_dict_to_str(self, input_dict, excl_keys = []):
                 output_str += "\n```\n"
         return output_str
 
-    def get_fpv_summary_results(self, result):
+    def get_formal_summary_results(self, result):
```

Comment:
Does s/formal_summary/summary/ work?

Reply:
Thanks Sri.. Yeah the naming is a bit awkward..